docs/How/release_process.md

```diff
@@ -24,6 +24,23 @@ The below diagram illustrates the life-cycle of a single branch and indicates th
     
     This is why we strongly discourage forking Project Mu for direct modification (ie. consumption, not contribution). Instead, leverage the distributed repo management system and override management system to integrate proprietary code/modules.
 
+## Current Branch Status
+
+While the earliest release branches may not be included in this process, starting with `release/201903` and going forwards the status of each branch will be recorded in the README.rst file at the root of the branch. In general, the README found in Basecore will contain information that is common to all of the Mu submodules, but each submodule will also have its own README for each release branch that contains notes specific to the development that occurs in that submodule during a release cycle.
+
+The README will also contain a summary of the branch status at a given time. For example, here is the status of Basecore `release/201903` as of the time of this writing:
+
+```text
+Status:
+  In Development
+
+Entered Development:
+  2019/03/25
+
+Anticipated Stabilization:

```

Comment:
Is this entry into Stabilization, or completion of Stabilization?  If both, do we assume we will always start/finish in 1 calendar month without crossing into another month?

To minimize text churn and improve consistency, perhaps consider...

Current Phase:  Active Development
Phase Entry Date:  2019-03-25
Phase Exit Date (tentative):  2019-05

Reply:
I was intending this to mean "enter stabilization", which is more of a process than an end-state (as evidenced by the number of RCs we already have in 1808). I'm happy to clarify if you think it'll help.